.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -89,7 +223,7 @@ jobs:
                 prNumber: '' + prData.num,
                 branchName: prData.branchName,
                 repo: prData.repo,
-                commit: prData.commit,
+                commit: process.env.COLLISION === 'false' ? prData.commit : '',
```

Comment:
To pass `commit: ''`, I think we need to update this line to prevent the error fixed by https://github.com/vitejs/vite/pull/18778.
https://github.com/vitejs/vite-ecosystem-ci/blob/6adc524597abdf2a0976c0ea2f681faa8632f5ea/.github/workflows/ecosystem-ci-from-pr.yml#L122

Reply:
yes, let me do so!

Reply:
here you go! https://github.com/vitejs/vite-ecosystem-ci/pull/339

Reply:
merged that, thanks!